pkgs/development/python-modules/dronecan/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "dronecan";
+  version = "1.0.25";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0WKmVZwE6OgBckWWvPcn5BYqXMEt6Mr1P68UMHfRp4I=";
+  };
+
+  doCheck = false;
```

Comment:
This was due to a very weird error that I was getting on tests, and since it has been manually tested to work for our use case for a while now I did not see the harm in disabling the tests, tho I should have noted that somewhere.

But I was able to fix the tests now, not sure if it is the ideal fix since I'm overriding the default checkPhase, but the tests run successfully now.